pytorch_lightning/accelerators/cpu_accelerator.py

```diff
@@ -53,10 +53,8 @@ def setup(self, model):
         self.trainer.model = model
 
     def train(self):
```

Comment:
why don't we define this function in `Accelerator`?

Reply:
good point :+1: